src/main/java/com/example/zoospringv1/ticketOffice/Ticket.java

```diff
@@ -0,0 +1,16 @@
+package com.example.zoospringv1.ticketOffice;
+
+public class Ticket {
+    int numberOfTickets;
+
+    public Ticket(final int numberOfTickets, final String ticketNormal, Currency pln) {
```

Comment:
czemu konstruktor dostaje parametry których w ogóle nie używa? jezeli planujesz to uzupełnić to polecam dopisać
// TODO uzupełnić o wykorzystanie parametrów ticketNormal i pln